### DISCUSSION TITLE
PR number and link missing in releases

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#1846] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/54838975/189474714-adb0ddcb-1de7-4f79-9596-35b28bf26a6d.png)
PR number and link is gone from releases from 9/9 (https://github.com/cataclysmbnteam/Cataclysm-BN/releases/tag/cbn-experimental-2022-09-09-225277). Is this an intentional change? In my opinion having them both helps a lot when navigating to corresponding PRs.

**Comment by Coolthulhu** [comment]
Not intentional, not sure what happened there.

**Comment by olanti-p** [comment, marked as answer]
Must be some change on GitHub side. PR numbers are still in the text, the website just doesn't show them.

Opening one of the older releases for editing and saving it without changes made the numbers disappear as well.
https://github.com/cataclysmbnteam/Cataclysm-BN/releases/tag/cbn-experimental-2022-09-03-1123

On another note, GitHub now also provides automatic changelog generation (at least from the web editor), tried it out on another release, looks ok
https://github.com/cataclysmbnteam/Cataclysm-BN/releases/tag/cbn-experimental-2022-09-09-2206
